### ISSUE TITLE
Repost message on fail

### ISSUE THREAD

**Issue by ghost** [madcowfred/GoPostStuff#18]
I could not find this anywhere. Does GPS automatically repost messages on failure?

I get a few warnings like this:

Post error: 441 Posting Failed. Message-ID is not unique E1

Thanks for the app and the help.

**Comment by e3m88** [comment]
Same error since a few days using a server, I guees we need to live with it and post more par2 blocks

**Comment by ghost** [comment]
Yeah, I increased par2's to 30%, up from 15%. I think that should cover it.
